apps/evans/components/pages/explore/AudiobooksHero.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 import cx from 'classnames';
 import { t } from '@friends-library/locale';
-import NextBgImage from 'next-bg-image';
+import NextBgImage, { bgColor } from 'next-bg-image';
 import Dual from '@/components/core/Dual';
 import HeadphonesImage from '@/public/images/headphones.jpg';
 import WaveformSVG from '@/public/images/waveform.svg';
 
 interface Props {
   numBooks: number;
   className?: string;
+  eager?: boolean;
 }
 
-const AudiobooksHero: React.FC<Props> = ({ className, numBooks }) => (
+const AudiobooksHero: React.FC<Props> = ({ className, numBooks, eager }) => (
   <NextBgImage
     className={cx(
       className,
       `text-center p-16 pb-48 md:pb-56 lg:pb-[18rem] overflow-hidden`,
     )}
-    src={HeadphonesImage}
+    eager={eager}
+    src={[HeadphonesImage, bgColor(`rgb(148,64,88)`)]}
```

Comment:
adding the color layer is just a little optimization so that on a slow network, as the main image is loading, they see a red background similar to the main image, so the transition is less jarring